examples/erc1155/Cargo.toml

```diff
@@ -0,0 +1,38 @@
+[package]
+name = "erc1155"
+version = "3.0.0-rc3"
+authors = ["Parity Technologies <admin@parity.io>"]
+edition = "2018"
+
+[dependencies]
+ink_primitives = { version = "3.0.0-rc3", path = "../../crates/primitives", default-features = false }
+ink_metadata = { version = "3.0.0-rc3", path = "../../crates/metadata", default-features = false, features = ["derive"], optional = true }
+ink_env = { version = "3.0.0-rc3", path = "../../crates/env", default-features = false }
+ink_storage = { version = "3.0.0-rc3", path = "../../crates/storage", default-features = false }
+ink_lang = { version = "3.0.0-rc3", path = "../../crates/lang", default-features = false }
+ink_prelude = { version = "3.0.0-rc3", path = "../../crates/prelude", default-features = false }
+
+scale = { package = "parity-scale-codec", version = "2.1", default-features = false, features = ["derive"] }
+scale-info = { version = "0.6", default-features = false, features = ["derive"], optional = true }
+
+[lib]
+name = "erc1155"
+path = "lib.rs"
+crate-type = ["cdylib"]
+
+[features]
+default = ["std"]
+std = [
+    "ink_primitives/std",
+    "ink_metadata",
+    "ink_metadata/std",
+    "ink_env/std",
+    "ink_storage/std",
+    "ink_lang/std",
+    "ink_prelude/std",
+    "scale/std",
+    "scale-info",
+    "scale-info/std",
+]
+ink-as-dependency = []
```

Comment:
probably for debugging with `debug_message!` you now want to propagate the `ink-debug` feature to `ink_env`.